eslint-config-angular/index.mjs

```diff
@@ -70,22 +74,11 @@ export const configBase = typescriptEslint.config({
     '@typescript-eslint/prefer-function-type': ['error'],
     '@typescript-eslint/unified-signatures': ['error'],
 
-    'array-bracket-spacing': ['error'],
-    'arrow-parens': ['error', 'as-needed'],
-    'arrow-spacing': ['error'],
-    'block-spacing': ['error'],
-    'curly': ['error'],
-    'jsdoc/newline-after-description': ['off'],
```

Comment:
Why?

We should actually switch to tsdoc here, but not in this MR

Reply:
Because this was turning off a rule that is no longer included in the default rules config.

Reply:
Removed in 2nd commit.